src/_includes/docs/swift-package-manager/migrate-macos-project-manually.md

```diff
@@ -0,0 +1,74 @@
+Once you enable the Swift Package Manager feature, the Flutter CLI will try
+to migrate your project to add Swift Package Manager integration. This makes
+your project download the Swift packages that your Flutter plugins depend on.
+
+However, the Flutter CLI may be unable to migrate your project if you've
+modified it significantly.
+
+Please [file a bug][] before migrating manually to help the Flutter team improve
+the automatic migration.
+Please include the error message you received and consider including a copy of
+the of the following files in your bug report:
+
+* `macos/Runner.xcodeproj/project.pbxproj`
+* `macos/Runner.xcodeproj/xcshareddata/xcschemes/Runner.xcscheme`
+  (or the xcscheme for the flavor used)
+
+### Step 1: Add FlutterGeneratedPluginSwiftPackage Package Dependency {:.no_toc}
+
+1. Open your app (`macos/Runner.xcworkspace`) in Xcode.
+2. Navigate to Package Dependencies for the project.
```

Comment:
Same edits as above. (Menu text should be bolded, for example.)

Reply:
Fixed!